cor_graphical/code/cor_graphics_violin_scatter.R

```diff
@@ -107,16 +107,19 @@ for (typ in c("line","violin")) {
       for (k in 1:length(trt)) {
         for (t in 1:length(times)) {
           
+          y.breaks <- seq(ifelse(plots[i] %in% c("bindSpike","bindRBD"), -1, 0), ifelse(plots[i] %in% c("bindSpike","bindRBD"), 6, 4), 1)
```

Comment:
Did you intend for these updates to be captured by this PR?

Reply:
Yes, sorry about the confusion. These are small format changes